cmd/node/main.go

```diff
@@ -540,6 +560,14 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 	}
 
 	log.Trace("creating nodes coordinator")
+	if ctx.IsSet(isNodefullArchive.Name) {
+		generalConfig.StoragePruning.FullArchive = ctx.GlobalBool(isNodefullArchive.Name)
+	}
+	if ctx.IsSet(numEpochsToSave.Name) {
```

Comment:
you have to add verification that these configs are correct, if the node is a validator than epochsToSave must be higher than 2.

Reply:
done. also changes some conditions. numActivePersisters should be set in all the cases